.github/workflows/release.yml

```
@@ -18,53 +18,53 @@ jobs:
       pull-requests: write # to create pull requests (semantic-release)
 
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v3
-      with:
-        fetch-depth: 0
-        # This is important - it ensures the action runs in the context of your repository
-        # and can push changes
-        token: ${{ secrets.GH_TOKEN }}
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)